test/selenium/specs/ts_all_specs.rb

```diff
@@ -6,7 +6,7 @@
 require_relative "tc_history"
 # Don't run the posting process tests on Chrome as the
 # posting process has changed.
-if @browser == "firefox" || @browser == "safari"
+if @browser == "firefox"
```

Comment:
Also, I assumed that these same tests would pass for Safari, but there seems to be an issue when these tests run on Safari. @sammyshj will look into it.